.github/workflows/superset-translations.yml

```diff
@@ -29,10 +29,6 @@ jobs:
         uses: ./.github/actions/cached-dependencies
```

Comment:
I think this file should be `frontend-translation` as it only does that (doesn't do backend)